developers/node-tutorial.md

```diff
@@ -425,6 +425,157 @@ when you created the wallet.
 
 With your wallet funded, you can move on to the next step.
 
+## Golang Client Library Guide {#golang-client-library}
```

Comment:
let's move this to `developers/golang-client-tutorial.md` or some similar title. i'd do it myself but want you to keep the credit 🫡